package.json

```diff
@@ -27,6 +27,6 @@
   "devDependencies": {
     "rimraf": "^2.2.6",
     "tape": "^4.6.0",
-    "tap-spec": "^4.1.1"
+    "tap-spec": "^5.0.0"
```

Comment:
Here's the [Changelog](https://github.com/scottcorgan/tap-spec/compare/4.1.1...5.0.0) - I don't think this will affect this project.